test/fixtures/transport/index.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
```

Comment:
can we test for an esm transport as well (just call it index.mjs)

Reply:
I'll take this action

Reply:
we are already testing an esm transport.